### PR TITLE
Minor enhancements for future use

### PR DESCRIPTION
**Pull request by MichaelColonel**
- Added calculation of transformation for one frame to another for IEC standard. For example transformation from Patient Support frame to Collimator frame: Patient Support -> Fixed Reference -> Gantry -> Collimator. That will be used in RoomsEyeView module for static RAS data and dynamic treatment models.
- Added busy cursor for DRR calculation process.

### CODE REVIEW

File: Beams/Logic/vtkSlicerIECTransformLogic.cxx [hunk 1]
```diff
@@ -401,7 +401,8 @@ vtkMRMLLinearTransformNode* vtkSlicerIECTransformLogic::GetTransformNodeBetween(
 }
 
 //-----------------------------------------------------------------------------
-bool vtkSlicerIECTransformLogic::GetTransformBetween(CoordinateSystemIdentifier fromFrame, CoordinateSystemIdentifier toFrame, vtkGeneralTransform* outputTransform)
+bool vtkSlicerIECTransformLogic::GetTransformBetween(CoordinateSystemIdentifier fromFrame, CoordinateSystemIdentifier toFrame, 
+  vtkGeneralTransform* outputTransform, bool transformForBeam)
```
Comment (cpinter) [inline Beams/Logic/vtkSlicerIECTransformLogic.cxx]: We usually indicate the default value here for readability purposes. Can you change add this?
`bool transformForBeam/*=true*/`

File: DrrImageComputation/qSlicerDrrImageComputationModuleWidget.cxx [hunk 2]
```diff
@@ -572,9 +572,14 @@ void qSlicerDrrImageComputationModuleWidget::onComputeDrrClicked()
     return;
   }
   
+  QApplication::setOverrideCursor(Qt::WaitCursor);
+
   bool result = d->logic()->ComputePlastimatchDRR( parameterNode, ctVolumeNode);
   if (result)
   {
+    // TODO: Restore previously selected node
```
Comment (cpinter) [inline DrrImageComputation/qSlicerDrrImageComputationModuleWidget.cxx]: It seems to me that the TODO was resolved so it can be removed.

**Comment by cpinter** [comment]
Thanks, looks good. I had only two minor comments.

**Comment by MichaelColonel** [comment]
I've changed these minor comments.

**Pull request merged**